save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -108,7 +108,7 @@ class DownloadProjectController(
         @RequestPart(required = true) executionRequest: ExecutionRequest,
         @RequestPart("fileInfo", required = false) fileInfos: Flux<FileInfo>,
         @RequestPart("file", required = false) files: Flux<FilePart>,
-    ): Mono<TextResponse> = Mono.just(ResponseEntity("Clone pending", HttpStatus.ACCEPTED))
+    ): Mono<TextResponse> = Mono.just(ResponseEntity("Clone pending, execution id is ${executionRequest.executionId}", HttpStatus.ACCEPTED))
```

Comment:
move "Clone pending, execution id is" to a constant

Reply:
Moved